src/DataCollector/SeoCollector.php

```diff
@@ -164,6 +168,11 @@ public function getTitle(): ?string
         return $this->data['title'];
     }
 
+    public function getTitleTruncate(): ?string
+    {
+        return (string) u($this->data['title'])->truncate(20, '...', false);
```

Comment:
Si tu le convertis en chaine tu auras tout le temps un `string` et jamais la valeur null

```suggestion
        return u($this->data['title'])->truncate(20, '...', false);
```

Reply:
même chose pour les methodes en dessous